TestDockerFilesystemStats fails on RHEL7 with HasFilesystem:false

HasFilesystem is expected to be true in the test

```
--- FAIL: TestDockerFilesystemStats (10.97s)
        Location:       docker_test.go:336
	Error:		Expected not to be nil.
	Messages:	got info: {Spec:{CreationTime:2016-02-27 03:58:05.94095891 +0000 UTC Aliases:[sharp_aryabhata 75c534be25bd074d7fdf9ae1f88698e8b389cf3fab7392c65023c1ea37f72b33] Namespace:docker Labels:map[] Envs:map[] HasCpu:true Cpu:{Limit:1024 MaxLimit:0 Mask:0-3 Quota:0 Period:0} HasMemory:true Memory:{Limit:18446744073709551615 Reservation:0 SwapLimit:18446744073709551615} HasCustomMetrics:false CustomMetrics:[] HasNetwork:true HasFilesystem:false HasDiskIo:true Image:busybox} Stats:[0xc2080561e0]}
		
FAIL
```

--- a/integration/tests/api/docker_test.go
+++ b/integration/tests/api/docker_test.go
@@ -295,6 +295,12 @@ func TestDockerFilesystemStats(t *testing.T) {
 	fm := framework.New(t)
 	defer fm.Cleanup()
 
+	storageDriver := fm.Docker().StorageDriver()
+	if storageDriver == framework.DeviceMapper {
+		// Filesystem stats not supported with devicemapper, yet
+		return
+	}
+
 	const (
 		ddUsage       = uint64(1 << 3) // 1 KB
 		sleepDuration = 10 * time.Second
@@ -307,7 +313,6 @@ func TestDockerFilesystemStats(t *testing.T) {
 		Count:  1,
 	}
 	needsBaseUsageCheck := false
-	storageDriver := fm.Docker().StorageDriver()
 	switch storageDriver {
 	case framework.Aufs, framework.Overlay:
 		needsBaseUsageCheck = true

